invokeai/app/services/invocation_queue.py

```diff
@@ -1,30 +1,17 @@
 # Copyright (c) 2022 Kyle Schouviller (https://github.com/kyle0654)
 
+import time
 from abc import ABC, abstractmethod
 from queue import Queue
-import time
+
+from pydantic import BaseModel, Field
 
 
-# TODO: make this serializable
-class InvocationQueueItem:
-    # session_id: str
+class InvocationQueueItem(BaseModel):
     graph_execution_state_id: str
     invocation_id: str
     invoke_all: bool
```

Comment:
This should probably be a field with a default (I think the rest should probably be fields for good measure too, though without defaults, since they require a value)

Reply:
Most things that can be a pydantic model probably should be, and their class attrs should all be fields. Besides pydantic providing runtime validation for the models, this is very useful for consumers of the api bc you get schemas for all the models.